app/models/repo.rb

```diff
@@ -70,7 +70,9 @@ def with_travis_service
   end
 
   def enabled_for?(checker)
-    Array(Settings.public_send(checker).enabled_repos).include?(name)
+    Array(Settings.public_send(checker).enabled_repos).each_with_object([]) { |value, values|
+      values << (value.kind_of?(Array) ? value.first.to_s : value)
+    }.include?(name)
```

Comment:
You can't do that as not all objects returned from `enabled_repos` are Enumerable. That was my first implementation :)

Code smell? `each_with_object` is just `reduce` yielding the memo value at the end of the block automatically. So you'd be saying that the higher order function `reduce` is a code smell... 😕